cluster/images/etcd/rollback/etcd237/pkg/fileutil/fileutil.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2015 CoreOS, Inc.
```

Comment:
@wojtek-t How shall we move this forward? Has mml verified this? Can you check again with him? Thank you!